mcserver/settings.py

```diff
@@ -239,13 +239,16 @@ def strip_sensitive_data(event, hint):
         send_default_pii=True
     )
 
-REDIS_URL = config('REDIS_URL')
+SQS_URL = config('SQS_URL')
```

Comment:
@antoinefalisse 
There is an important moment with replacing Redis with SQS. SQS is a good solution for the queue, but it is not enough for our needs. We use Celery for running scheduled tasks. So, we also use `celery beat` for running tasks on schedule. Instead of using a simple file for managing the schedule (that usually works badly with container-based platforms, like ECS, due to some limitations), we use an alternative scheduler, `celery-redbeat` (check the option `CELERY_BEAT_SCHEDULER`).

So, even after replacing Redis with SQS for the queue broker, we still need Redis to manage the schedule.
Actually, for the tasks we have in the project (very simple tasks), we do not need such a robust solution as SQS. Redis works great both for queue and scheduler. So my idea was to use one service instead of two.

There are two possible solutions:

1. To use SQS as a broker. But in that case, as I understand, we also need to add `celery[sqs]` to the requirements - https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/sqs.html Also, we still need to keep Redis for the scheduler.
2. Do not use SQS, but use Redis instead for both - https://aws.amazon.com/redis/

What do you think&

Reply:
@olehkorkh-planeks thanks for the detailed explanation. I had misunderstood that we could just swap Redis by SQS. I agree with using only one service instead of two, and your option 2 therefore sounds better. Let's go for that one and disregard this PR as well as [this one](https://github.com/stanfordnmbl/opencap-infrastructure/pull/1)